rust-runtime/aws-smithy-types/src/date_time/mod.rs

```diff
@@ -596,4 +601,50 @@ mod test {
         assert!(fourth > third);
         assert!(fourth == fourth);
     }
+
+    #[cfg(all(
+        test,
+        aws_sdk_unstable,
+        feature = "serde-deserialize",
+        feature = "serde-serialize"
+    ))]
+    #[test]
+    fn human_readable_datetime() {
+        use serde::{Deserialize, Serialize};
+
+        let datetime = DateTime::from_secs(1576540098);
+        #[derive(Serialize, Deserialize, PartialEq)]
+        struct Test {
+            datetime: DateTime,
+        }
+        let datetime_json = r#"{"datetime":"2019-12-16T23:48:18Z"}"#;
+        assert!(serde_json::to_string(&Test { datetime }).ok() == Some(datetime_json.to_string()));
+
+        let test = serde_json::from_str::<Test>(&datetime_json).ok();
+        assert!(test.is_some());
+        assert!(test.unwrap().datetime == datetime);
+    }
+
+    /// checks that they are serialized into tuples
+    #[cfg(all(
+        test,
+        aws_sdk_unstable,
+        feature = "serde-deserialize",
+        feature = "serde-serialize"
+    ))]
+    #[test]
+    fn not_human_readable_datetime() {
+        let cbor = ciborium::value::Value::Array(vec![
```

Comment:
Should there also be a Cargo.toml change that adds the ciborium crate in this PR?

Reply:
I think ciborium got lost somewhere when I was separating it into sub-PR. Let me fix.